src/web/modules/transactions/update/update.http.ts

```diff
@@ -174,9 +209,20 @@ export async function update(req: Request, res: Response): Promise<void> {
 
   try {
     const data = await validateAndAddDefaults(req.file.buffer.toLocaleString(), req.user)
+
+    const transformedData:Dictionary<string|number>[] = data.map((row) => {
```

Comment:
never used Dictionary , but looks fine

Reply:
This a lodash type, which is handy when just need a type temporarily so you pass it into somewhere else.